services/port_manager/src/main/java/com/futurewei/alcor/portmanager/repo/PortRepository.java

```diff
@@ -116,18 +116,19 @@ public void createPortAndNeighborBulk(List<PortEntity> portEntities, Map<String,
             //Add neighborInfos to neighborCache
             if (neighbors != null) {
                 for (Map.Entry<String, List<NeighborInfo>> entry: neighbors.entrySet()) {
-                    PortNeighbors portNeighbors = neighborCache.get(entry.getKey());
```

Comment:
btw, Please add neighborCache= cacheFactory.getCache(PortNeighbors.class);  to the constructor, Thanks!

Reply:
Sure. Fixed in https://github.com/futurewei-cloud/alcor/pull/263/commits/a758e1848834c30726ea0c395dea2542c34dc66b

Reply:
hello @chenpiaoping It seems that there are some corner cases to be covered by UTs. Could you please add those UTs when you get a chance? Thanks.

Reply:
ok